src/winter_onboarding_2021/fleet_management_service/views/cab.clj

```diff
@@ -0,0 +1,31 @@
+(ns winter-onboarding-2021.fleet-management-service.views.cab)
+
+(defn add-cab-form [data]
```

Comment:
See if you can extract components from this form. For example: LabelledTextInput. 

This will allow reusing the components in the future, and reusing components would give the UI more consistent feel overall.

Reply:
Doing this

I am also thinking to rename `add-cab-form` to `cab-form` because we will be reusing this form on edit cab page